event_contact/models/event.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2016 OpenSynergy Indonesia
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class event_event(models.Model):
+    _inherit = 'event.event'
+
+    contacts_ids = fields.Many2many(
+        string='Contacts',
+        comodel_name='res.partner',
+        relation='event_contacts_rel',
```

Comment:
Do you really need to specify the relation table? Is it not enough with the automatic name?